### DISCUSSION TITLE
OSD no relative alt

### DISCUSSION THREAD

**Discussion by diphtong** [svpcom/wfb-ng#168] [Questions]
Hi,
wifibroadcast_osd is not showing relative alt, but qgc does.
All other data is shown properly.
Maybe anybody has an idea?

Thank you!

![Bildschirmfoto vom 2021-10-01 14-50-02](https://user-images.githubusercontent.com/91046964/135622696-eb4fe4dd-0c5e-427c-ad2b-653caa4f3908.png)

**Comment by svpcom** [comment]
Add debug print to osd_mavlink.c -> parse_mavlink_packet -> osd_alt = mavlink_msg_vfr_hud_get_alt(&msg);

**Comment by diphtong** [comment]
Hello,
thank you for your answer!

But how exactly have i to do this?
I have to paste "mavlink_msg_vfr_hud_get_alt(&msg);" somewhere in osdmavlink.c, right?
